ngraph/frontend/onnx/onnx_import/src/op/leaky_relu.cpp

```diff
@@ -21,11 +21,8 @@ namespace ngraph
                     auto data = node.get_ng_inputs().at(0);
                     double alpha = node.get_attribute_value<double>("alpha", 0.01);
 
-                    CHECK_VALID_NODE(
-                        node, alpha >= 0 && alpha <= 1, " alpha value should be in range (0,1)");
-
                     std::shared_ptr<ngraph::Node> alpha_node =
-                        default_opset::Constant::create(data.get_element_type(), Shape{}, {alpha});
+                        default_opset::Constant::create(data.get_element_type(), Shape{1}, {alpha});
```

Comment:
Before the constant was a scalar, now it is a constant with size {1}. Why do we need it?
If we are talking about check. Do we have a model with such values or spec requirements?

Reply:
@ilyachur It is a part of OnnxImporter-MO parity. Legacy MO produces `PRelu` with `slope's shape={1}` and it is expected in MO layer tests.